assets/scripts/dialogs/SaveAsImageDialog.jsx

```diff
@@ -295,7 +295,7 @@ class SaveAsImageDialog extends React.Component {
                     />
                   </a>
                 ) : (
-                  <button disabled>
+                  <button disabled={true}>
```

Comment:
For HTML elements like `<button>` (not React components) did the rule also require `disabled={true}` to be explicitly set? Will it complain if the value is `disabled="true"`?

The reason why I ask is in HTML normally `disabled="false"` is not false. The presence of the `disabled=` attribute makes it disabled (regardless of its value). The only way to make its value _actually_ false is to remove the attribute.

Alterrnatively, does React or JSX already adjust for this behavior to be consistent so we don't need to worry about it?

Reply:
Good questions! That one stuck out to me too. 

It did trigger the rule, since its JSX and I set the rule to 'always'. Setting ```disabled="true"``` led to no complaints.

The second question is an interesting edge case! Since it's React, we're using JSX and the DOM API so I reckon it would work.

Reply:
This is fun. Setting ```disabled={false}``` does lead to a render html of `<button>`, which (luckily for us), makes the snapshot test fail for that component: 

![snapshot](https://user-images.githubusercontent.com/10539855/82846634-748bb600-9eaf-11ea-9110-08280683802b.png)

Reply:
Interesting side note: using ```disabled={true}``` or ```disabled`` both lead to output in the snapshot like ```disabled=" "``` which is good enough for [HTML](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute). 

Setting ```disabled="true"``` Does give you a friendly dev message in the console: 
>  Received the string `true` for the boolean attribute `disabled`. Although this works, it will not work as expected if you pass the string "false". Did you mean disabled={true}?

Reply:
Ok, it seems that JSX does the right thing for us when we use boolean attributes for HTML elements, which is all we need to know!